pkg/collector/corechecks/servicediscovery/events.go

```diff
@@ -0,0 +1,132 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package servicediscovery
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/DataDog/datadog-agent/comp/core/hostname"
+	"github.com/DataDog/datadog-agent/comp/core/hostname/hostnameimpl"
+	"github.com/DataDog/datadog-agent/comp/forwarder/eventplatform"
+	"github.com/DataDog/datadog-agent/pkg/aggregator/sender"
+	pkgconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+type eventType string
+
+const (
+	eventTypeStartService     = "start-service"
+	eventTypeEndService       = "end-service"
+	eventTypeHeartbeatService = "heartbeat-service"
+)
+
+type eventPayload struct {
+	ApiVersion          string    `json:"api_version"`
```

Comment:
It should be OK to not include the API version and request type in the payload since they are already included in the event. The spec in https://docs.google.com/document/d/1dc5KRIJjtONFjkWaITZF4i53UHgAjuextT6jyWmzJ-k/edit#heading=h.6ytopxdl5w22 went through a few iterations and when we added these fields to the outer struct, we might have forgotten to remove them from the inner one.

Reply:
I still see these fields in the document (at least in the example). I can remove these fields, but just double-checking.